tests/EndToEndBuildTests/BasicProvider/BasicProvider.DesignTime/BasicProvider.DesignTime.fsproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <FSharpToolsDirectory>typeproviders</FSharpToolsDirectory>
+    <DefineConstants>NO_GENERATIVE</DefineConstants>
+    <DefineConstants>IS_DESIGNTIME</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- It is normal to include a copy of the runtime helpers in the design-time DLL, to help build the code model which gets mapped to target files -->
+    <Compile Include="..\BasicProvider\BasicProvider.Runtime.fs">
+      <Link>BasicProvider.Runtime.fs</Link>
+    </Compile>
+ 
+     <!-- The design-time component contains a copy of the TPSDK core files -->
+    <Compile Include="..\..\ProvidedTypes\ProvidedTypes.fsi">
+      <Link>..\..\ProvidedTypes\ProvidedTypes.fsi</Link>
```

Comment:
Nit: `<Link />` isn't necessary if it's the same value as the `Include` attribute.